populate_db.py

```diff
@@ -210,16 +210,21 @@ def create_permissions():
     mod = Role.query.get(4)
     attend = Role.query.get(5)
     regist = Role.query.get(6)
+    ownr = Role.query.get(7)
```

Comment:
is this a typo? @shreyanshdwivedi

Reply:
No @uds5501. Actually I've tried to make the name similar to what we are following for other roles. Like `orgr` for organizer